File: reliability/log_generator.rb
class LogGenerator
  SERVICES = %w{
    web
    admin
    api
  }.freeze

  def self.sample(uuid)
    service = SERVICES.sample

    [
      "id=#{uuid}",
      "service_name=#{service}",
      "process=#{service}.#{rand(1..4000)}",
      "sample#load_avg_1m=#{rand.round(3)}",
      "sample#load_avg_5m=#{rand.round(3)}",
      "sample#load_avg_15m=#{rand.round(3)}",
    ].join(" ")
  end
end

File: data/lib/log_generator.py
import random

def sample(uuid):
  service = random.choice(["web", "admin", "api"])
  
  return " ".join([
    f'id={uuid}',
    f'service_name={service}',
    f'process={service}.{random.randint(1, 4001)}',
    f'sample#load_avg_1m={round(random.random(), 3)}',
    f'sample#load_avg_5m={round(random.random(), 3)}',
    f'sample#load_avg_15m={round(random.random(), 3)}'
  ])
